core/src/player.rs

```diff
@@ -1380,30 +1380,21 @@ impl Player {
                 // and fires "drag" events. Other objects are ignored.
                 if context.input.is_mouse_down() {
                     context.mouse_over_object = new_over_object;
```

Comment:
By the way, this feels odd, shouldn't this value be updated regardless of mouse input? I might just not be seeing something here, so correct me if I'm wrong.